snuba/migrations/runner.py

```diff
@@ -36,6 +32,18 @@
 DIST_TABLE_NAME = "migrations_dist"
 
 
+def get_active_migration_groups() -> List[MigrationGroup]:
```

Comment:
Please return a `Sequence` instead of a `List`.
This is mainly because List is a mutable type, the callsites should not need a mutable structure. Sequence is immutable.
The callsite also should not need anything more specific than a Sequence for this (otherwise they would likely be doing something wrong)

Reply:
Thank you for pointing this out; I modified.